### DISCUSSION TITLE
[Vue warn]: setup() return property "$auth" should not start with "$" or "_" which are reserved prefixes for Vue internals

### DISCUSSION THREAD

**Discussion by fathi91** [nuxt/nuxt#16638] [Questions]
Good day,
i'm getting this warning for my auth plugin `[Vue warn]: setup() return property "$auth" should not start with "$" or "_" which are reserved prefixes for Vue internals`

any idea why is that? all nuxt plugins get the  "$" automatically when they get injected

**Comment by paulvonber** [comment]
That's quite old, but I have exactly the same problem, but with .client.vue component while using any plugin in it
nuxt v3.4.3
